go.mod

```diff
@@ -18,6 +18,6 @@ replace (
 	github.com/docker/cli => github.com/docker/cli v25.0.4-0.20240221083216-f67e569a8fb9+incompatible
 	github.com/docker/compose/v2 => github.com/docker/compose/v2 v2.24.6
 	github.com/docker/scout-cli => github.com/docker/scout-cli v1.4.1
-	github.com/moby/buildkit => github.com/moby/buildkit v0.13.0-beta3.0.20240201135300-d906167d0b34
+	github.com/moby/buildkit => github.com/moby/buildkit v0.13.0
```

Comment:
I think buildkit is a special case and we always want latest changes from master, not a specific release.

Reply:
updated with `@master`